### PR TITLE
SL-248: types adjustments

### PR DESCRIPTION
**Pull request by chris-miaskowski**
**Summary**

- some minor pluralization and fields rename

### CODE REVIEW

File: http.d.ts [hunk 1]
```diff
@@ -111,14 +111,14 @@ export interface IHttpLink {
 export interface IHttpRequestBody {
   description?: string;
   required?: boolean;
-  content: IHttpContent[];
+  contents: IHttpContent[];
```
Comment (StefanDywersant) [inline http.d.ts]: Not sure whether word "contents" means the same as `content[]`. Some native will need to comment on this ;)

File: http.d.ts [hunk 2]
```diff
@@ -111,14 +111,14 @@ export interface IHttpLink {
...
 
 export interface IHttpRequest {
-  path?: IHttpPathParam[];
-  query?: IHttpQueryParam[];
-  headers?: IHttpHeaderParam[];
-  cookie?: IHttpCookieParam[];
+  pathParams?: IHttpPathParam[];
+  queryParams?: IHttpQueryParam[];
+  headerParams?: IHttpHeaderParam[];
+  cookieParams?: IHttpCookieParam[];
```
Comment (StefanDywersant) [inline http.d.ts]: Mixed feelings. It is more descriptive, but we have typed language with IDE support for telling us what's hidden behind fields. But yes, still more descriptive ;)

Comment (pytlesk4) [reply http.d.ts]: I think it is too descriptive, like @StefanDywersant said the IDE does this for you.

File: http.d.ts (hunk 1 above)
Comment (pytlesk4) [reply http.d.ts]: This makes sense to me!

File: http.d.ts (hunk 2 above)
Comment (marbemac) [reply http.d.ts]: Don't have a strong opinion either way, leave it to you folks!

**Pull request merged**